templates/static/rollup.config.js

```diff
@@ -6,7 +6,7 @@ import { terser } from 'rollup-plugin-terser';
 export default {
 	input: 'index.js',
 	output: {
-		file: 'dist/core.min.js'
+		file: 'dist/core.min.js'		
```

Comment:
you may want to set up your IDE to automatically fix these...
```suggestion
		file: 'dist/core.min.js'
```